client/src/components/OfflineWarning/offline-warning.tsx

```diff
@@ -32,7 +36,10 @@ function OfflineWarning({
   }
 
   return showWarning ? (
-    <div className='offline-warning alert-info'>{t('misc.offline')}</div>
+    <>
+      <div className='offline-warning alert-info'>{message}</div>
+      <div style={{ height: `38px` }} />
```

Comment:
If this is acting as a spacer, please use the `Spacer` component we have.

Reply:
I am not entirely sure if it can be replaced. Since the offline warning was changed into fixed position it doesn't occupy space on page anymore, so I just copied the approach that was used for Flash warning as I'm not very familiar with React yet.

https://github.com/freeCodeCamp/freeCodeCamp/blob/af4ead336aa223e5e3733771a8d03e5c88d37479/client/src/components/Flash/index.tsx#L53-L57

Please tell me if Spacer should be used instead after all and I will redo it.

Reply:
So I've tried using spacer and since it's 31px high this was the result

![image](https://user-images.githubusercontent.com/64734725/134966054-3f2df06a-491a-44ca-8cca-7febc6db5dbf.png)